conftest.py

```diff
@@ -0,0 +1,13 @@
+from mypy.test.helpers import PytestSuite
+import inspect
+import pytest
+
+def pytest_addoption(parser):
+    parser.addoption('--update-testcases', action='store_true',
+                     dest='UPDATE_TESTCASES')
+
+def pytest_pycollect_makeitem(collector, name, obj):
+     if (inspect.isclass(obj) and issubclass(obj, PytestSuite) and
+            obj is not PytestSuite):
```

Comment:
Can you avoid the `and obj is not PytestSuite` part here? A special case like this always makes me a little uneasy that there's a bug, or at least a gotcha in the API, hiding somewhere.

Ah, in fact I think what it would take to make this work cleanly without that condition is just to add a `cases` method right on the `PytestSuite` base class that just returns an empty list. Will continue in an inline comment over there for better context.